Loop/Managers/LoopDataManager.swift

```diff
@@ -184,12 +199,13 @@ class LoopDataManager {
 
      - parameter resultsHandler: A closure called once the values have been retrieved. The closure takes the following arguments:
         - predictedGlucose:     The calculated timeline of predicted glucose values
+        - insulinOnBoard        Current insulin on board
```

Comment:
The order here doesn't match the arguments below (which is OK for Swift 2, but not for Swift 3)